code/script.js

```diff
@@ -1,25 +1,36 @@
 // Variables that point to selected DOM elements
-const chat = document.getElementById('chat')
+const chat = document.getElementById('chat');
```

Comment:
I havent fully gotten the hang of the DOM. I added five links here - send-btn, name-form etc.. but your code seems to work just fine without having them listed up here